insoundz_api/src/enhancer.py

```diff
@@ -152,7 +152,7 @@ def _enhancement_finish(
         else:
             self._logger.exception(f"[{sid}] Unexpected status {status}")
 
-    def _enhancement_start(self, api, src, dst, retention, preset, pbar):
+    def _enhancement_start(self, api, src, dst, retention, preset, enable_stereo, pbar):
```

Comment:
I think it's always better to add new arguments to the end. Otherwise it can break old implementaion.
For example if someone calls the old omplemetion of enhance_file with enhance_file(65, 'flat', '3).